util/weekly_build/README

```diff
@@ -0,0 +1 @@
+Concretization for deployments should be done with `spack concretize --fresh` so as to ensure that the stack would concretize the same way if the build cache weren't there.
```

Comment:
Wondering if we should use something else than weekly, because I plan to do this nightly

Reply:
Yeah it would make sense to use a more generic term... "scheduled_builds"? "automated_builds"? "build_cacher"? :shrug:

Reply:
We can figure this out later, let's leave it as is for now.